app/src/main/java/org/xbmc/kore/ui/MoviesActivity.java

```diff
@@ -67,10 +80,15 @@ protected void onCreate(Bundle savedInstanceState) {
 
             // Setup animations
             if (Utils.isLollipopOrLater()) {
-                movieListFragment.setExitTransition(null);
+                //Fade added to prevent shared element from disappearing very shortly at the start of the transition.
+                Fade fade = new Fade();
+                fade.setDuration(50);
+                movieListFragment.setExitTransition(fade);
```

Comment:
You will see that the shared element will still be visible in the fading view. By making the fade shorter it is not noticeable. The trick I used in MovieDetailsFragment by making the poster invisible does not seem to work here.